cypress/integration/utils/sharedTests/navbar.ts

```diff
@@ -54,7 +54,7 @@ const mobileNavbarClickFlow = newClickFlow(
     newExpectation(
       "should show DoltHub links",
       "[data-cy=mobile-navbar-links] > li",
-      newShouldArgs("be.visible.and.have.length", 7),
+      newShouldArgs("be.visible.and.have.length.of.at.least, 7"),
```

Comment:
7 shouldn't be in the string
```suggestion
      newShouldArgs("be.visible.and.have.length.of.at.least", 7),
```